src/test/java/com/puppycrawl/tools/checkstyle/checks/NewlineAtEndOfFileCheckTest.java

```diff
@@ -67,6 +79,25 @@ public void testNewlineLfAtEndOfFile() throws Exception {
     }
 
     @Test
+    public void testCloseRandomAccessFile() throws Exception {
```

Comment:
we recently decided to force code with mocks to have javadoc comment explaining a reason why it impossible to make functional test. This case is approved to me based on mock, lets just left a message to future maintainers/contributors.
please add javadoc comment on method.